build.gradle.kts

```diff
@@ -1,4 +1,4 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id("com.android.application") version "8.1.1" apply false
+    id("com.android.application") version "7.3.1" apply false
```

Comment:
Do we really need to downgrade the plugin version?

Reply:
I did this so that my version of AS could work correctly with this project)

PS: yes, I admit, it was not necessary to change the plugin version, 
but to update AS, but that would have taken time)